### ISSUE TITLE
organize data path

### ISSUE THREAD

**Issue by riberaborrell** [riberaborrell/sde-importance-sampling#4]
one data directory per choice of parameters. The directories should follow this tree structure:

- settings of the overdamped langevin equaiton
-- data directory 
--- potential-name
---- alpha_i (parameters of the potential)
----- beta
------ target-set type (hypercube at [1, 3]^n)

- hjb solution
-- settings directory
--- 'hjb-solution'
---- h

- MC Sampling
-- settings directory
--- 'mc-sampling'
---- dt
----- N

- Importance sampling
-- parent directory (hjb_dir_path, model_dir_path, soc_dir_path)
--- 'is'
---- dt
----- N

- Metadynamics
-- settings directory
--- dt
---- sigma_i
----- k
------ N

- Function approximator
-- settings directory 
--- parametrization (value-f, control)
---- nn name (two-layer-nn, gaussian-ansatz)
----- nn rel path (nn hyperparameters, for example inner layer dimension)
------ initialization (random, null, meta, hjb) 

- Gaussian ansatz functions
-- distributed (uniform, meta)
--- theta (null, meta, optimal)
---- m_i 
----

- SGD
-------- gd ipa (gradient descent by using infinitessimal perturbation analysis)
--------- lr (learning rate)

- [x] get_settings_dir_path(): absolute path
- [x] get_hjb_solution_dir_path(): absolute path
- [x] get_mc_dir_path(): absolute path
- [x] get_is_dir_path(): absolute path
- [x] get_metadynamics_dir_path(): absolute path
- [x] get_nn_function_approximator_dir_path(): absolute path
- [x] get_ansatz_rel_dir_path(): relative path
- [x] get_is_dir_path(): absolute path
- [x] get_som_dir_path(): absolute path
